assets/scss/_registry.scss

```diff
@@ -1,138 +1,24 @@
-.registry {
```

Comment:
The use of these styles are nowhere to be found, including in the commit that introduced them:

- #247